ledger/src/entry_notifier_interface.rs

```diff
@@ -8,4 +8,4 @@ pub trait EntryNotifier {
     fn notify_entry(&self, slot: Slot, index: usize, entry: &EntrySummary);
 }
 
-pub type EntryNotifierLock = Arc<RwLock<dyn EntryNotifier + Sync + Send>>;
```

Comment:
Please remove the unused RwLock to make sure clippy is happy

Reply:
done!